parlai/core/build_data.py

```diff
@@ -21,7 +21,11 @@
 import tqdm
 import math
 import zipfile
-from multiprocessing import Pool
+
+try:
+    from torch.multiprocessing import Pool
```

Comment:
using torch.multiprocessing fixes the issues around spawn that I was originally fighting, hence the changes in the import method.